Feature/add docs for cookies (#661)
grafana/k6-docs: src/data/markdown/docs/05 Examples/01 Examples/08 cookies-example.md

</CodeGroup>

## Delete a cookie in the VU cookie jar

> To delete a cookie in the jar by specifying url and name
> you'd do something like this:

Reviewer: ```suggestion
To delete a cookie in the jar for a specific URL and name, use the `delete` method.
```

  jar.set('https://httpbin.test.k6.io/cookies', 'my_cookie_1', 'hello world_1');
  jar.set('https://httpbin.test.k6.io/cookies', 'my_cookie_2', 'hello world_2');

  const res1 = http.get('https://httpbin.test.k6.io/cookies');
  check(res1, {
    'res1 has status 200': (r) => r.status === 200,
    "res1 has cookie 'my_cookie_1'": (r) => r.json().cookies.my_cookie_1 !== null,
    'res1 cookie has correct value_1': (r) => r.json().cookies.my_cookie_1 == 'hello world_1',
    "res1 has cookie 'my_cookie_2'": (r) => r.json().cookies.my_cookie_2 !== null,
    'res1 cookie has correct value_2': (r) => r.json().cookies.my_cookie_2 == 'hello world_2',
  });

  jar.delete('https://httpbin.test.k6.io/cookies', 'my_cookie_1');

  const res2 = http.get('https://httpbin.test.k6.io/cookies');
  check(res2, {
    'res2 has status 200': (r) => r.status === 200,
    "res2 hasn't cookie 'my_cookie_1'": (r) => r.json().cookies.my_cookie_1 == null,
    "res2 has cookie 'my_cookie_2'": (r) => r.json().cookies.my_cookie_2 !== null,
    'res2 cookie has correct value_2': (r) => r.json().cookies.my_cookie_2 == 'hello world_2',
  });
}
```

</CodeGroup>

## Clear all cookies in the VU cookie jar

> To clear all cookies in the jar by specifying url
> you'd do something like this:

Reviewer: ```suggestion
To clear all cookies in the jar by specifying url, use the `clear` method.
```
